portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/login")
+public class LoginServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html");
+    String urlToRedirectTo = "/index.html";
+    UserService userService = UserServiceFactory.getUserService();
+    if (userService.isUserLoggedIn()) {
+      String userEmail = userService.getCurrentUser().getEmail();
+      
+      String logoutUrl = userService.createLogoutURL(urlToRedirectTo);
+
+      response.getWriter().println(logoutUrl);
+    } else {
+      String loginUrl = userService.createLoginURL(urlToRedirectTo);
+
+      response.getWriter().println(loginUrl);
+    }
```

Comment:
I think this can be consolidated as follows:
```suggestion
    String url = "";
    if (userService.isUserLoggedIn()) {
      String userEmail = userService.getCurrentUser().getEmail();
      url = userService.createLogoutURL(urlToRedirectTo);
    } else {
      url = userService.createLoginURL(urlToRedirectTo);
    }
    response.getWriter().println(url);
```